opencog/cython/opencog/atomspace.pxd

```diff
@@ -88,6 +88,7 @@ cdef extern from "opencog/atomspace/Handle.h" namespace "opencog":
     cdef cppclass cHandle "opencog::Handle":
         cHandle()
         cHandle(UUID)
+        cHandle(void*)
```

Comment:
I should say that semantically, if Cython supported reference parameters, this would be:

```
cHandle(AtomPtr&)
```
and not
```
cHandle(void*)
```
because that is what is happening behind the scenes.